tests/component_tests/test_session.py

```diff
@@ -162,15 +157,16 @@ def init_primary(self):
             'test': {
                 'default_schema': 'schema_alias_error',
                 'schemas': {
-                    'schemas': ['schema_alias_error'],
+                    'test_schema': 'schema_alias_error',
                     'schema_alias_error': 'unknown_schema'
                 }
             }
         }
         ru.write_json(facility_cfg, '%s/resource_facility.json' % user_cfg_dir)
-        with self.assertRaises(KeyError):
-            with mock.patch.object(Session, '_init_primary', new=init_primary):
-                Session()
```

Comment:
this test suppose to catch non-existing alias reference (see comment above)

Reply:
Same - remove

Reply:
Done